config/wpcalypso.json

```diff
@@ -179,6 +179,7 @@
 		"themes/text-search-lots": true,
 		"themes/assembler-first": true,
 		"themes/tiers": false,
+		"two-factor/enhanced-security": true,
```

Comment:
Are we intentionally not enabling this on staging where more a12s could test it?

Reply:
Just an oversight, thanks for noticing it! :) - https://github.com/Automattic/wp-calypso/pull/85234/commits/8a964d79bc4b01e71b4a0647a9417b3fb0789157